_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-tactile
```

Comment:
This snuck in again, if you're happy to remove it. I'll see if I can figure out where it's coming from. As I mentioned today I caused the file to be created when I tried to use github pages but as far as I'm aware I've turned everything github-pages related off.

This is some info on the file: [About GitHub Pages and Jekyll - GitHub Help](https://help.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll)

Hopefully this might become a non-issue when we move the repo, although I think settings come along for the ride, like everything else does. Worth a shot anyway.

Reply:
I've removed the file from master, which is where this came from I guess.